### PR TITLE
Update pulsar kerberos document

### PR DESCRIPTION
**Pull request by zymap**
Currently, kerberos document is lack of some description when using client to connect brokers with kerberos.

### CODE REVIEW

**Comment by zymap** [comment]
@jiazhai  PTAL

File: site2/docs/security-kerberos.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ It is also a choice to have 2 separate JAAS configuration files: the file for br
  - Set `authenticationProviders` to choose `AuthenticationProviderSasl`;
  - Set `saslJaasClientAllowedIds` regex for principal that is allowed to connect to broker. 
  - Set `saslJaasBrokerSectionName` that corresponding to the section in JAAS configuration file for broker.
+ - Set `superUserRoles` that corresponding to the name registered in kdc.
```
Comment (jiazhai) [inline site2/docs/security-kerberos.md]: This need set when `authorizationEnabled` is set to true.
How about adding it at around line 270, which is related to authorization

File: site2/docs/security-kerberos.md [hunk 2]
```diff
@@ -156,6 +158,18 @@ The following is an example of creating a Java client:
 
 Make sure that the keytabs configured in the `pulsar_jaas.conf` file and kdc server in the `krb5.conf` file are reachable by the operating system user who is starting pulsar client.
 
+If you are using command line, you can continue with these step:
```
Comment (jiazhai) [inline site2/docs/security-kerberos.md]: Thanks for the fix

File: site2/docs/security-kerberos.md (hunk 1 above)
Comment (zymap) [reply site2/docs/security-kerberos.md]: updated. PTAL

File: site2/docs/security-kerberos.md [hunk 3]
```diff
@@ -271,6 +283,13 @@ saslJaasBrokerSectionName=PulsarBroker
 
 For Kerberos authentication, the authenticated principal is used as the role token for Pulsar authorization.  For more information of authorization in Pulsar, see [security authorization](security-authorization.md).
 
+If you enabled authorizationEnabled you need set `superUserRoles` that corresponding to the name registered in kdc
```
Comment (jiazhai) [inline site2/docs/security-kerberos.md]: set `superUserRoles`  in` broker.conf`

Comment (zymap) [reply site2/docs/security-kerberos.md]: updated

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
run java8 tests

**Pull request merged**